test/e2e.test.ts

```diff
@@ -0,0 +1,69 @@
+/// <reference path="../node_modules/@types/mocha/index.d.ts" />
+
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+import * as ethers from "ethers";
+import * as dotenv from "dotenv";
+
+import { Config, Deposit, SubConfig } from "../src/types";
+import * as helpers from "../src/helpers";
+import { createInstance } from "../src";
+
+chai.use(chaiAsPromised.default);
+const expect = chai.expect;
+dotenv.config();
```

Comment:
nit: dotenv docs warn that you should put `dotenv.config()` as early as you can in your file to avoid weird behavior. Could move this up a couple lines.